src/pages/get-involved.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Layout from '../components/Layout';
+import ContributorShowcase from '../components/ContributorShowcase';
+
+const getInvolvedPage = () => (
+  <Layout>
+    <h1>Get Involved</h1>
+    <div className="markdown">
+      <ul>
+        <li>
+          <a href="https://opencollective.com/openclimatefix">Donate</a>
+          {' '}
+          to our OpenCollective
+        </li>
+        <li>
+          Follow us by
+          {' '}
+          <a href="https://eepurl.com/guCjvH">signing up to our newsletter</a>
+        </li>
+        <li>
+          Join the
+          {' '}
+          <a href="https://openclimatefix.discourse.group/">OCF discussion forum</a>
+        </li>
+      </ul>
+      <p>
+        Also check out
+        {' '}
+        <a href="/blog/2019-03-06-how-to-get-involved">
+          Jack&apos;s blog post about getting involved
+        </a>
+        .
+      </p>
+      <h2>Contribute to one of our repositories on Github</h2>
+      <p>
+        Check out our
+        {' '}
+        <a href="https://github.com/search?l=&p=1&q=user%3Aopenclimatefix+label%3A%22good+first+issue%22&ref=advsearch&type=Issues&utf8=%E2%9C%93&state=open">List of Good First Issues</a>
+        to get started!
```

Comment:
need another space here after the link, i.e. `{' '}`

Reply:
just fixed

Reply:
jeeez so quick! Amazing, thanks!!

Reply:
Yeah it just popped up in my email